rs/tests/testnets/io_perf_benchmark.rs

```diff
@@ -91,10 +93,80 @@ fn main() -> Result<()> {
     Ok(())
 }
 
+fn switch_to_ssd(log: &Logger, hostname: &str) {
+    let script = r##"
+        #!/bin/bash
+        set -e
+        exec 2>&1
+
+        # Get the name of the currently running VM.
+        # virsh prints four lines for a single instance running, two lines of header,
+        # VM info and trailing empty line. The script should fail if the format is
+        # different or there are multiple VMs running. The trailing line is lost when
+        # assigning to a variable.
+        virsh_list=$(sudo virsh list)
+        if [ $(echo "$virsh_list" | wc -l) -ne 3 ]; then
+                echo "Unexpected virsh list output"
+                exit 2
+        fi
+        VMNAME=$(echo "$virsh_list" | awk '{ if (NR==3) print $2 }')
+
+        # Shutdown the VM
+        echo "Shutting down $VMNAME"
+        for i in {1..300}; do
+                if [ $(sudo virsh list | grep $VMNAME | wc -l) -eq 0 ]; then
+                        break
+                fi
+                echo Waiting for shutdown of $VMNAME: retry $i...
+                sleep 1
+                sudo virsh shutdown $VMNAME || true
+        done
+
+        # Get the file name and dd it to disk device
+        CONFIG=$(mktemp)
+        trap "rm -f $CONFIG" INT TERM EXIT
+        sudo virsh dumpxml $VMNAME > $CONFIG
+        IMAGE="$(xmlstarlet sel -t -v "string(/domain/devices/disk[target[@dev='vda']]/source/@file)" "$CONFIG")"
+        echo "Moving $VMNAME to /dev/hostlvm/guest"
+
+        # Patch the config to point to the disk device
+        xmlstarlet ed --inplace -a "//domain/devices/disk[target[@dev='vda']]/driver" -t attr -n discard -v unmap "$CONFIG"
+        xmlstarlet ed --inplace -a "//domain/devices/disk[target[@dev='vda']]/driver" -t attr -n cache -v none "$CONFIG"
+        xmlstarlet ed --inplace -u "//domain/devices/disk[target[@dev='vda']]/@type" -v block "$CONFIG"
+        xmlstarlet ed --inplace -d "//domain/devices/disk[target[@dev='vda']]/source/@file" "$CONFIG"
+        xmlstarlet ed --inplace -a "//domain/devices/disk[target[@dev='vda']]/source" -t attr -n dev -v "/dev/hostlvm/guestos" "$CONFIG"
+        sudo dd if=$IMAGE of=/dev/hostlvm/guestos status=progress bs=512MiB oflag=direct iflag=direct
+
+        sudo virsh create $CONFIG
+        echo "Migration done"
+    "##;
+
+    if std::env::var("SSH_AUTH_SOCK") == Err(std::env::VarError::NotPresent) {
+        panic!("No $SSH_AUTH_SOCK vairable provided");
+    }
+    let mut ssh = Command::new("ssh")
+        .arg("-o")
+        .arg("StrictHostKeyChecking=no")
+        .arg("farm@".to_owned() + hostname)
+        .arg(script)
+        .stdout(Stdio::piped())
+        .spawn()
+        .unwrap();
+    for l in BufReader::new(ssh.stdout.as_mut().unwrap()).lines() {
+        info!(&log, "SSH {} out: {}", hostname, l.unwrap());
+    }
+
+    let ssh_status = ssh.wait().unwrap();
+    if !ssh_status.success() {
+        panic!("SSH to {} failed with: {}", hostname, ssh_status);
+    }
```

Comment:
For all other SSH interactions under `rs/tests` we use the [ssh2](https://crates.io/crates/ssh2) crate (through the [SshSession](https://sourcegraph.com/github.com/dfinity/ic@6fd184fef85ec1344b5d0d2c96a83ba72bfacd83/-/blob/rs/tests/driver/src/driver/test_env_api.rs?L1555) trait. I'm not saying we shouldn't use the `ssh` command but it's a bit unusual. But if it works it works ;)

Reply:
True. My comment was more about using the `ssh2`. But feel free to keep using the `ssh` command. It's not that important.